src/components/tooltip/NetworkMarker.js

```diff
@@ -69,6 +69,21 @@ function _getHumanReadableDataStatus(status: NetworkStatus): string {
   }
 }
 
+function _getHumanReadableHttpVersion(httpVersion: string): string {
+  switch (httpVersion) {
+    case 'h3':
+      return 'HTTP/3.0';
+    case 'h2':
+      return 'HTTP/2.0';
+    case 'http/1.0':
+      return 'HTTP/1.0';
+    case 'http/1.1':
+      return 'HTTP/1.1';
```

Comment:
I think that we don't need this `HTTP/` part, given `HTTP` is already in the label. What do you think?

Reply:
I think this is a good point, I will update the patch.